### DISCUSSION TITLE
Welcome to Spiel-Info-Q11-2022 Discussions!

### DISCUSSION THREAD

**Discussion by Jonas-a11y** [Jonas-a11y/Spiel-Info-Q11-2022#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sc0rp10n0419** [comment]
Hallo

**Reply by Jonas-a11y** [reply]
Moin

**Reply by sc0rp10n0419** [reply]
Was machst du?

**Reply by Jonas-a11y** [reply]
Antworten

**Reply by sc0rp10n0419** [reply]
Ich auch

**Reply by Jonas-a11y** [reply]
Du meintest dass es einen fehler bei deinen schüssen gibt nh? Wo bzw. was ist der?
